dotnet/samples/Demos/CodeInterpreterPlugin/README.md

```diff
@@ -0,0 +1,33 @@
+# Semantic Kernel - Code Interpreter Plugin with Azure Container Apps
+
+This example demonstrates how to do AI Code Interpretetion using a Plugin with Azure Container Apps to execute python code in a container.
+
+## Configuring Secrets
+
+The example require credentials to access OpenAI and Azure Container Apps (ACA)
+
+If you have set up those credentials as secrets within Secret Manager or through environment variables for other samples from the solution in which this project is found, they will be re-used.
+
+### To set your secrets with Secret Manager:
+
+```
+dotnet user-secrets init
+
+dotnet user-secrets set "OpenAI:ApiKey" "..."
+
+dotnet user-secrets set "AzureContainerApps:BearerKey" " .. current token .. "
```

Comment:
I am curious what is the bearer key is for? The dotnet plugin has an auth callback, correct?

Reply:
It does and the demo uses the callback to provide the token from the configuration. In a normal scenario this would be handled by the user on acquiring the token.

Reply:
Changed the provider the use Azure.Identity to get the token and to make this closer to a real world scenario.